libcontainer/factory_linux.go

```diff
@@ -168,6 +168,9 @@ func (l *LinuxFactory) Create(id string, config *configs.Config) (Container, err
 	if err := os.MkdirAll(containerRoot, 0700); err != nil {
 		return nil, newGenericError(err, SystemError)
 	}
+	if err := syscall.Mkfifo(filepath.Join(containerRoot, execFifoFilename), 0655); err != nil {
```

Comment:
`0655` is a pretty strange permission set.  I'm not clear on the intended auth here, but the signal-based approach was limited by kill(2)'s `CAP_KILL` and/or UID requirements (although SIGCONT had a same-session special case).  I expect you want `0600` here to match that.